Examples/VoiceMemos/VoiceMemos/SceneDelegate.swift

```diff
@@ -19,8 +19,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
           environment: VoiceMemosEnvironment(
             audioPlayer: .live,
             audioRecorder: .live,
-            date: Date.init,
-            mainQueue: .main,
+            mainRunLoop: .main,
```

Comment:
simplifies things a bit to just us a run loop instead of a queue and a date initializer.